recipes/openssl/3.x.x/conandata.yml

```diff
@@ -1,10 +1,13 @@
 sources:
   3.0.5:
     url: https://www.openssl.org/source/openssl-3.0.5.tar.gz
+    url: https://www.openssl.org/source/old/3.0/openssl-3.0.5.tar.gz
```

Comment:
This is not necessary anymore for 3.x versions, only for 1.x versions

They promised to not move these files anymore so that permanent valid download URLs exist

Reply:
OK, I reverted it.